### ISSUE TITLE
Checking LMFDB for broken external links

### ISSUE THREAD

**Issue by jenpaulhus** [LMFDB/lmfdb#6225]
Today someone noted on a bug report form that we had a broken external link for a paper referenced in a knowl.  I fixed the particular example (an msri.org file that is now located at slmath.org) but it would be nice to have a system that occasionally checks the lmfdb for broken external links.

**Comment by edgarcosta** [comment]
We already have a tool for that (except, it never went through the knowls), but at some point, we decided to disable them from the GitHub workflow tests, as many of the external links were not reliable, so we had too many false test failures because of these.

https://github.com/LMFDB/lmfdb/blob/main/lmfdb/tests/__init__.py#L60-L75

**Comment by AndrewVSutherland** [comment]
I don't think we want to do this as part of our GitHub/CI workflow, but I do think it would be useful to have a test we could run on demand that would also cover knowls.

**Comment by roed314** [comment]
@havarddj, any thoughts on what direction to take this?

**Comment by havarddj** [comment]
I thought about this for a bit, and perhaps the simplest would be to run a cron job once a month/quarter/year, and email one of the editors with a list of the dead links in the knowls.  I don't really feel like it's necessary to spin up a whole github action for this.
It could also be possible to email the person who last edited the knowl, but I'm not sure that's ideal.

**Comment by havarddj** [comment]
Is the point of this to say that we should check these extra links as well? It's a little hard to extract the intention from all the text.

**Comment by roed314** [comment]
Yep; I removed them from the CI since they were failing so often, and just wanted to remark here that they should be added to whatever we come up with.  Sorry for the wall of text!

**Comment by havarddj** [comment]
No problem! Just wondering what was going on.
